packages/mesa.rb

```diff
@@ -26,49 +18,34 @@ class Mesa < Package
   depends_on 'elfutils'
   depends_on 'llvm'
 
-  def self.patch
-    system "wget -O- https://patchwork.freedesktop.org/patch/257229/raw | patch -Rp1"
-  end
-
   def self.build
     system "pip3 uninstall -y Mako MarkupSafe || :"
     system "pip3 install --prefix \"#{CREW_PREFIX}\" --root \"#{CREW_DEST_DIR}\" Mako==1.0.7"
     system "pip3 install --prefix \"#{CREW_PREFIX}\" Mako==1.0.7"
-    case ARCH
-    when 'i686'
-      gallium_drivers = 'i915,nouveau,pl111,svga,swrast,vc4,virgl'
-      dri_drivers = 'i915,i965,nouveau,radeon,r200,swrast'
-    when 'x86_64'
-      gallium_drivers = 'i915,nouveau,r300,r600,radeonsi,pl111,svga,swrast,swr,vc4,virgl'
-      dri_drivers = 'i915,i965,nouveau,radeon,r200,swrast'
-    when 'aarch64', 'armv7l'
-      gallium_drivers = 'nouveau,r300,freedreno,pl111,swrast,tegra,vc4,virgl'
-      dri_drivers = 'nouveau,radeon,r200,swrast'
-    end
-    system "./configure",
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           "--enable-shared-glapi",
-           "--with-gallium-drivers=#{gallium_drivers}",
-           "--with-dri-drivers=#{dri_drivers}",
-           "--enable-osmesa",
-           "--enable-opengl",
-           "--enable-egl",
-           "--enable-xvmc",
-           "--disable-asm",
-           "--enable-nine",
-           "--enable-gles1",
-           "--enable-gles2",
-           "--with-platforms=x11,drm,wayland",
-           "--enable-gbm",
-           "--enable-xa",
-           "--enable-glx",
-           "--enable-dri",
-           "--enable-llvm"
-    system "make"
+    system "meson",
+      "--prefix=#{CREW_PREFIX}",
+      "--libdir=#{CREW_LIB_PREFIX}",
+      "-Dshared-glapi=true",
+      "-Dgallium-drivers=auto",
+      "-Ddri-drivers=auto",
```

Comment:
Good call.  This is a lot easier than explicitly listing all the drivers.  However, I'm still a bit squeamish about this update since it affects other packages that have this as a dependency.  Please test other packages to make sure they still work as expected after installing this update when you get a chance.  Namely, the following:
```
cairo.rb:  depends_on 'mesa'
jasper.rb:  depends_on 'mesa'
libepoxy.rb:  depends_on 'mesa'
libglu.rb:  depends_on 'mesa'
libsdl.rb:  depends_on 'mesa'
libwebp.rb:  depends_on 'mesa'
sommelier.rb:  depends_on 'mesa'
wxwidgets.rb:  depends_on 'mesa'
xkbcomp.rb:  depends_on 'mesa'
```